tests/scripts/all.sh

```diff
@@ -119,6 +119,7 @@ pre_initialize_variables () {
     MEMORY=0
     FORCE=0
     KEEP_GOING=0
+    CTEST_OUTPUT_ON_FAILURE=1
```

Comment:
That won't get you far unless you `export` it.

Reply:
Correct, it doesn't do anything. Will add in separate PR.